src/style.css

```diff
@@ -1 +1,45 @@
-/* add styles here */
+:root {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-size: cover;
```

Comment:
try to fix all tests.
looks like tests expects star to be centered. use background-position property for it